src/main/scala/tech/cryptonomic/conseil/tezos/NodeSenderOperations.scala

```diff
@@ -0,0 +1,333 @@
+package tech.cryptonomic.conseil.tezos
+
+import tech.cryptonomic.conseil.config.{BatchFetchConfiguration, SodiumConfiguration}
+import tech.cryptonomic.conseil.generic.rpc.DataFetcher
+import tech.cryptonomic.conseil.tezos.TezosTypes.{
+  blockHeadHash,
+  Account,
+  AccountId,
+  AppliedOperation,
+  Block,
+  BlockData,
+  BlockHash,
+  ForgedOperation,
+  InjectedOperation,
+  ManagerKey,
+  MichelsonExpression,
+  OperationsGroup,
+  ProtocolId
+}
+import tech.cryptonomic.conseil.util.CryptoUtil
+import tech.cryptonomic.conseil.util.CryptoUtil.KeyStore
+import tech.cryptonomic.conseil.util.JsonUtil.{fromJson, toJson, JsonString}
+import tech.cryptonomic.conseil.generic.rpc.RpcHandler
+import scala.util.Try
+import cats.Functor
+import cats.syntax.functor._
+import cats.syntax.flatMap._
+import cats.syntax.try_._
+
+object NodeSenderOperations {
```

Comment:
Please document this object. Also, could we call it something like `NodeWriteOperations`? It's not clear why a node is being sent. :grin:

Reply:
Sure... 😄